packages/graphql_server/src/processRepo.ts

```diff
@@ -173,3 +173,10 @@ export const parseGitHubUrl = (url: string) => {
 
   return null
 }
+
+export const exposeProjectsData = async () => {
```

Comment:
a resolver function should not be placed in processRepo, create a new folder `resolver` with `resolver/projects.ts`
Place the resolver function there, and log errors as shown in the backend channel with `context`, so they are mounted to the actual request

Reply:
Could you show me an example to use context in this file? I could not find any example in the project.

Reply:
https://jst-ss23.slack.com/archives/C0530GLH79D/p1686771046878509